### PR TITLE
opentyrian, fix references to libSDL*, don't use -Werror

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: games-arcade/opentyrian/patches/opentyrian-2.1.20130907.patchset [hunk 1]
```diff
@@ -41,5 +41,34 @@ index 3cffa49..d0f85a4 100644
...
++ # BUILD SETTINGS ###########################################
+ 
+ ifneq ($(filter Msys Cygwin, $(shell uname -o)), )
+     PLATFORM := WIN32
+@@ -22,7 +22,7 @@ OBJS := $(SRCS:src/%.c=obj/%.o)
+ # FLAGS ####################################################
+ 
+ ifneq ($(MAKECMDGOALS), release)
+-    EXTRA_CFLAGS += -g3 -O0 -Werror
++    EXTRA_CFLAGS += -g3 -O0
```
Comment (alaviss) [inline games-arcade/opentyrian/patches/opentyrian-2.1.20130907.patchset]: I think you fixed the wrong thing. from what I can see here, setting `MAKECMDGOALS` to `release` (exporting that in the recipe should do) will disable `-Werror`.

File: games-arcade/opentyrian/opentyrian-2.1.20130907.recipe [hunk 2]
```diff
@@ -41,12 +41,14 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
+	export MAKECMDGOALS=release
 	make $jobArgs
```
Comment (korli) [inline games-arcade/opentyrian/opentyrian-2.1.20130907.recipe]: didn't "make $jobArgs release" work?

Comment (Begasus) [reply games-arcade/opentyrian/opentyrian-2.1.20130907.recipe]: didn't try that :)

Comment (Begasus) [reply games-arcade/opentyrian/opentyrian-2.1.20130907.recipe]: it did ;)

**Pull request merged**